install_fathom.sh

```diff
@@ -64,7 +64,7 @@ After=network.target
 
 [Service]
 Type=simple
-User="$USER"
+User=$USER
```

Comment:
What problem is this solving? When I ran this on my server it produced `User=mitch` and that works.